src/components/movie-item/movie-item.tsx

```diff
@@ -36,9 +36,9 @@ export function MovieItem(props: any): React.ReactElement {
 
             <div className="movie-item-detail">
                 <div className="movie-item-detail-row">
-                    <span className="movie-item-name">{props.movie.name}</span>
+                    <span id={'movie-item-name-'+props.movie.name} className="movie-item-name">{props.movie.name}</span>
```

Comment:
id should be unique across the app, movie name does not suit

Reply:
fixed